### PR TITLE
Some changes for DOM manipulation

### PR DESCRIPTION
**Pull request by sh19910711**
The main changes are two things like the followings:

1. Use `class` attribute instead of `id` attribute (_inner_console_markup.html.erb)
2. ~~Change styles of the `resizer` element~~
  - ~~Use `position: absolute`~~
  - ~~Change its `background-color` to the dark-grey~~

~~Well, the absolute positioning is to add more elements like the `resizer` element. For example, I think to add "minimize button", "close button" and "setting button" to the console, and the absolute positioning is able to show multiple elements like that to the top of the console element.~~
(I will move this to another pull request)

### CODE REVIEW

File: lib/web_console/templates/error_page.js.erb [hunk 1]
```diff
@@ -51,21 +51,25 @@ function changeSourceExtract(frameId) {
 // better user experience, won't hurt.
 document.addEventListener('DOMContentLoaded', function() {
   var consoleElement = document.getElementById('console');
-  var resizerElement = document.getElementById('resizer')
+  var resizerElement = consoleElement.getElementsByClassName('resizer')[0];
   var containerElement = document.getElementById('container');
 
+  function getConsoleHeight() {
+    return consoleElement.offsetHeight + resizerElement.offsetHeight;
+  }
```
Comment (sh19910711) [inline lib/web_console/templates/error_page.js.erb]: This function is needed by the absolute positioning

File: lib/web_console/templates/style.css.erb [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
-#console div.console-inner { font-family: monospace; font-size: 11px; height: 100%; overflow: auto; background: #333; border: 0; padding: 0; margin: 0; }
+#console { position: fixed; left: 0; bottom: 0; width: 100%; height: 148px; overflow: none; padding: 0; margin: 0; background: none repeat scroll 0% 0% #333; z-index: 9999; }
+#console div.console-inner { font-family: monospace; font-size: 11px; height: 100%; overflow: auto; background: #333; border: 0; padding: 0 3px; margin: 0; }
 #console div.console-prompt-box { color: #FFF; }
 #console pre.console-message { color: #1AD027; margin: 0; border: 0; white-space: pre-wrap; background-color: #333; padding: 0; }
 #console div.console-focus span.console-cursor { background: #FEFEFE; color: #333; font-weight: bold; }
-#console div#resizer { width: 100%; height: 4px; cursor: ns-resize; }
-#console input#clipboard { height: 0px; padding: 0px; margin: 0px; width: 0px; margin-left: -1000px; }
+#console div.resizer { position: absolute; width: 100%; background: #666; height: 4px; margin-top: -4px; cursor: ns-resize; }
```
Comment (sh19910711) [inline lib/web_console/templates/style.css.erb]: > margin-top: -4px
Uhm..., I will change this prop.

**Comment by gsamokovarov** [comment]
> For example, I think to add "minimize button", "close button" and "setting button" to the console,

Great, go ahead. Anything else needed here before merging?

File: lib/web_console/templates/console.js.erb [hunk 3]
```diff
@@ -112,13 +113,18 @@ REPLConsole.prototype.install = function(container) {
...
-  this.clipboard = document.getElementById('clipboard');
+  this.inner = this.findChild('console-inner');
+  this.clipboard = this.findChild('clipboard');
   this.remotePath = container.dataset.remotePath;
   this.newPromptBox();
   this.insertCss();
 };
 
+// Find an element by class name
+REPLConsole.prototype.findChild = function(className) {
```
Comment (gsamokovarov) [inline lib/web_console/templates/console.js.erb]: Does it make sense to inline this function, like you did with `getConsoleHeight`? We can "promote" it to prototype function (instance method) once we have other functions needing it.

Comment (sh19910711) [inline lib/web_console/templates/console.js.erb]: > Anything else needed here before merging?
Right. I want to add little change before that.

> Does it make sense to inline this function
Yes, I will change it.

**Comment by sh19910711** [comment]
I think I will move the styling changes into a separate pull request. And now this change does not affect the looks and design.

@gsamokovarov I changed the `REPLConsole.prototype.findChild` to a inline function. Could you check it again? Thanks :)

**Comment by sh19910711** [comment]
@gsamokovarov Thanks for merging!

**Pull request merged**